tests/integration/cattletest/core/test_container.py

```diff
@@ -20,7 +19,8 @@ def test_container_create_count(admin_client, sim_context):
         assert c.state == 'running'
 
 
-def test_container_create_only(admin_client, sim_context):
+def test_container_create_only(admin_client, internal_test_client,
```

Comment:
You ultimately return a container that was last reloaded with the internal_test_client context. This test is called by other tests. You might want to take care to return a container scoped to the admin_client.

Could use @ibuildthecloud's input on this.